packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/cwts-on-signup-password.js

```diff
@@ -70,6 +80,7 @@ class CWTSOnSignupPasswordGroupingRule extends BaseGroupingRule {
     return !!(
       subject &&
       subject.email &&
+      'multiService' in subject &&
```

Comment:
Is it safer to do `in` than testing `subject.multiService`?

Reply:
subject.multiService can be empty or false, as long as it's passed in, I'm happy.